plugins/emmc/fu-emmc-device.c

```diff
@@ -205,6 +205,15 @@ fu_emmc_device_probe(FuDevice *device, GError **error)
 	fu_device_build_instance_id(device, NULL, "EMMC", "NAME", NULL);
 	fu_device_set_name(device, tmp);
 
+	/* firmware version */
+	tmp = g_udev_device_get_sysfs_attr(udev_parent, "fwrev");
+	if (tmp != NULL) {
+		fu_device_set_version_format(device, FWUPD_VERSION_FORMAT_NUMBER);
+		fu_device_set_version(device, tmp);
+	}
+	fu_device_add_instance_strsafe(device, "REV", tmp);
+	fu_device_build_instance_id(device, NULL, "EMMC", "NAME", "REV", NULL);
```

Comment:
Works for me, but both will need adding to the README.md in the emmc plugin directory.

Reply:
done